lib/PrintAsClang/ModuleContentsWriter.cpp

```diff
@@ -796,16 +796,18 @@ class ModuleWriter {
       }
     }
 
-    if (!delayedMembers.empty()) {
```

Comment:
Would it make sense to rename `delayedMembers` to something like `objcDelayedMembers` to make it clear that those are not emitted in non-objc mode?

Reply:
Good idea!